Release 4.1.0 (#156)
Concordium/concordium-java-sdk: CHANGELOG.md

- Added support for Encrypted Transfer with Memo.
- Added support for Configuring a baker.
- Added support for Configuring the account as a Delegator.
- A new native dependency has been introduced in order to support the new transaction types mentioned above,

>   so one will need to `make` the native dependencies again before building the java project.

Reviewer: ```suggestion
  As a result native dependencies need to be rebuilt before building the Java project.
```
